src/main/java/com/datasonnet/commands/Run.java

```diff
@@ -20,7 +20,8 @@
 
 @CommandLine.Command(
         name = "run",
-        description = "Transform data using DataSonnet"
+        description = "Transform data using DataSonnet",
+        footer = {"Available input and output formats are JSON, XML, and CSV"}
```

Comment:
Ha, that's where you can build this string using getPluginId() instead of hard-coding it. Get a list of available plugins from DataFormatService and iterate over them and get plugins ID of each one.

Reply:
I wish, unfortunately String arguments to annotations cannot be dynamic! I need to dig in and see if there's a good dynamic way to provide the footer.